INSTALL.md

```diff
@@ -6,101 +6,84 @@ Instructions for installing, testing and running *Glow*.
 
 ### One-Liner on Linux, macOS, WSL
 
-Long story short, you can install *Glow* with the following one-liner:
+You can install *Glow* with this one-liner:
 
     curl -L https://glow-lang.org/install/glow-install | sh
 
-This will make the command `glow` available to you,
-though you may have to restart your shell, or
-copy/paste a configuration command that will be suggested to you.
+This will make the command `glow` available to you, though you may have
+to restart your shell, or copy/paste a configuration command that will
+be suggested. The installation script requires `curl`, `sudo`, and `xz`.
```

Comment:
Is `xz` required? I can't find it in the install script.

Reply:
Yes, the `nix` installer uses it.